### ISSUE TITLE
[Request] Open Explorer Shell Context Menu

### ISSUE THREAD

**Issue by daniel-richter** [lin-ycv/EverythingPowerToys#38]
**Is your feature request related to a problem? Please describe.**
In Everything, a right clicking a file/folder in the results opens up the Windows Explorer context menu. It would be nice to have this possibility in EverythingPowerToys, too. 

**Describe the solution you'd like**
Does not have to be a right click. A new context menu button to open the Windows Explorer shell context menu would be great.

**Additional context**
E.g. the code provided in [CodeProject: Explorer Shell Context Menu - Add the Windows Explorer Shell Context Menu to your application](https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu) is very easy to integrate:
```cs
ShellContextMenu shellContextMenu = new ShellContextMenu();
FileInfo[] fileInfos = new[] { FileInfo(path) };
shellContextMenu.ShowContextMenu(fileInfos , this.PointToScreen(new Point(this.button1.Left,this.button1.Top)));
```

_Open questions_:
- The parameter `((Control.ModifierKeys & Keys.Shift) != 0 ? CMF.EXTENDEDVERBS : 0)` (calling `QueryContextMenu`) checks whether [Shift] is pressed and displays the extended context menu. Maybe displaying the extended context menu as default would be a nicer solution.
- Context menu icon: _More_ or _ButtonMenu_?

**Comment by lin-ycv** [comment]
to be clear, you're imagining something like this?
![image](https://user-images.githubusercontent.com/60565737/205946797-198d3580-2929-4052-b008-9315c2079853.png)

**My thoughts**
I'm not sure how practical this is? isn't run meant as a launcher for quickly finding and opening items?
an expanded context menu like this feels out of place IMO, opening the containing folder and right clicking doesn't really take that much more time.

Also, as seen from the screenshot, there's now quite a few context menu icons, taking up half the space, not sure how it'll display on lower res displays.

If really necessary, I think implementing it as right click would be the best option, but not sure if that'd raise any conflicts with PTR, will need some testing.

**Regarding your questions**
On windows 10 I don't really see any difference between extended and normal context menu, just an extra option for copy path?

for icons, I think More is more logical, Menu looks out of place
if we're completely fine with misusing icons, GlobalNavigationButton looks the best IMO
![image](https://user-images.githubusercontent.com/60565737/205950650-edb0e629-21be-41ac-83c8-84e51baa062c.png)

**Comment by lin-ycv** [comment]
This'll take some time to implement, there's over a thousand lines of code in [CodeProject: Explorer Shell Context Menu](https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu)
I don't think all of them are needed.

Also I'd like to clean up my own code first before adding more complicated codes.

**Comment by daniel-richter** [comment]
> This'll take some time to implement, there's over a thousand lines of code in [CodeProject: Explorer Shell Context Menu](https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu) I think some simplifications might be possible.

Yeah, but I'm afraid a lot of the code is neccessary. If you want, I could clean it up (would be ~650 loc).

**Comment by lin-ycv** [comment]
Sure, if you want, you can also implement it and do a PR

**Comment by lin-ycv** [comment]
I've not been successful in integrating this. Seeing as it's been 6 months since there's any updates on this, I can only assume no one else has been able to do it either.
If someone is able to implement this feature, please do a PR, i'm closing this for now.
